Controller/MediaAdminController.php

```diff
@@ -64,7 +65,7 @@ public function browserAction()
         $formView = $datagrid->getForm()->createView();
 
         // set the theme for the current Admin Form
-        $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
+        $this->setFormTheme($formView, $this->admin->getFilterTheme()[0]);
```

Comment:
and I get the first element of the array because the getFilterTheme return an array now. but the setFormTheme expect a string.